android-client-sdk/src/main/java/com/devcycle/sdk/android/api/EventQueue.kt

```diff
@@ -26,7 +26,7 @@ internal class EventQueue constructor(
     internal val aggregateEventMap: HashMap<String, HashMap<String, Event>> = HashMap()
 
     private val scheduler = Scheduler(coroutineScope, flushInMs)
-    private lateinit var scheduleJob: Job
```

Comment:
Do we not want to keep the lateinit and set nullable or do they conflict?

Reply:
I don't think the lateinit applies here because, based on the exception, it's accessed before it's initialized.

Reply:
Ah true